scipy/linalg/tests/test_matmul_toeplitz.py

```diff
@@ -0,0 +1,103 @@
+"""Test functions for linalg.matmul_toeplitz function
+"""
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from scipy.linalg import toeplitz, matmul_toeplitz
+
+import pytest
+from pytest import raises as assert_raises
+from numpy.testing import assert_allclose
+np.random.seed(42)
```

Comment:
This seeding should be moved to the start of every individual test case. Note that tests may be run out of order and in parallel.

Reply:
For preference, something like the following instead of using the `np.random.*` functions.

```python
class ToeplitzTestCase:

    def setup_method(self):
        self.rng = np.random.RandomState(42)

    def test_real(self):
        ...
        c = self.rng.normal(size=n)
        ...
```

Reply:
As suggested, I added the `setup_method`.